tesseract/tesseract_kinematics/include/tesseract_kinematics/ikfast/impl/ikfast_inv_kin.hpp

```diff
@@ -191,6 +191,17 @@ const std::vector<std::string>& IKFastInvKin::getJointNames() const { return joi
 const std::vector<std::string>& IKFastInvKin::getLinkNames() const { return link_names_; }
 const std::vector<std::string>& IKFastInvKin::getActiveLinkNames() const { return active_link_names_; }
 const tesseract_common::KinematicLimits& IKFastInvKin::getLimits() const { return limits_; }
+
+void IKFastInvKin::setLimits(tesseract_common::KinematicLimits limits)
```

Comment:
You would still need to call update at this point though right?

Reply:
I don't think so since we are setting them directly. I image the update will be just for rebuilding if needed.